test/acceptance/features/requirements.txt

```diff
@@ -2,4 +2,7 @@ behave==1.2.6
 requests==2.27.1
 polling2==0.5.0
 PyYAML==6.0
-semver==2.13.0
\ No newline at end of file
+semver==2.13.0
+cryptography
+kubernetes
+pyyaml
```

Comment:
I think this is a duplicate; we already have `PyYAML` pulled in as a dependency.  Can you check?

Reply:
you're right!